nvOC_miner_update.sh

```diff
@@ -786,15 +786,15 @@ sleep 2
 function compile-TPccminer {
   echo "Compiling tpruvot ccminer"
   echo " This could take a while ..."
-  if [[ $CUDA_VER == "8" ]]
+  if [[ $CUDA_VER == "8.0" ]]
   then
     get-sources TPccminer
-    cd ${NVOC_MINERS}/TPccminer/{TPccminer_src_hash_ver_9}
+    cd ${NVOC_MINERS}/TPccminer/src_cuda_8
```

Comment:
The sources folder for any version of the same miner is the same, you just need to call

get-sources ${NVOC_MINERS}/TPccminer src abcdfabcdf123451235abcdf

with correct hash depending on cuda version. You can exploit the usual unification pattern such that you can always have the correct hash under ${!xhash}

Reply:
Uhm, maybe the effort of the unified source hash variable is not worth, do what you think it's easier.

Reply:
Can do both, was just waiting to see how your get-sources gonna work. 
So you say there is gonna be one source folder for each miner?
` ${NVOC_MINERS}/TPccminer/src`
Then what happens to miners like tpruvot and klaust?
Which src version we gonna add as sub-module to its folder? Or we gonna remove src sub-module from repo, and just add it to update script?

Reply:
I think its a better idea to remove src submodule from repo and any one wants to re-compile get it downloaded.